clouddriver-cloudrun/src/main/java/com/netflix/spinnaker/clouddriver/cloudrun/model/CloudrunMetadataLabels.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2023 OpsMx, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.cloudrun.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class CloudrunMetadataLabels {
+
+  @JsonProperty("cloud.googleapis.com/location")
+  private String cloudGoogleapisComLocation;
+
+  public String getCloudGoogleapisComLocation() {
+    return cloudGoogleapisComLocation;
+  }
+
+  public void setCloudGoogleapisComLocation(String cloudGoogleapisComLocation) {
+    this.cloudGoogleapisComLocation = cloudGoogleapisComLocation;
+  }
```

Comment:
Can we use Lombok?

Reply:
@mattgogerly Please can you review the changes now?

Reply:
@mattgogerly Please can you review the changes?

Reply:
@mattgogerly Please can you review the changes?